src/module10/TestExeption.java

```diff
@@ -0,0 +1,7 @@
+package module10;
+
+public class TestExeption extends Exception {
+    public TestExeption(String message) {
+        super(message);
+    }
```

Comment:
Write a constructor for this class that takes a String argument and stores it inside the object with a String reference. Write a method that prints out the stored String